src/transactions/TransactionMeta.cpp

```diff
@@ -407,7 +407,6 @@ OperationMetaBuilder::setLedgerChangesFromSuccessfulOp(
             {
                 changes.emplace_back(LEDGER_ENTRY_CREATED);
                 changes.back().created() = *le;
-                changes.back().created().lastModifiedLedgerSeq = ledgerSeq;
```

Comment:
Is this an observable change?

Reply:
It shouldn't be, given that we now set the lastModifiedLedgerSeq when upserting entries into the thread map (this was the fix to the previous lastModifiedLedgerSeq bug).